### PR TITLE
{tools}[GCCcore/10.2.0] Maude v3.1

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

* [x] #13906
* [x] #13907
* [x] #13908

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/e915b9d79684eeb96b16168e45d79094 for a full test report.

File: easybuild/easyconfigs/m/Maude/Maude-3.1-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+# Building in parallel sometimes errors
+maxparallel = 1
+
+runtest = 'check'
+
+modextravars = {'MAUDE_LIB': '%(installdir)s/share'}
+
+sanity_check_paths = {
+    'files': ['share/%s.maude' % x for x in ['file', 'linear', 'prelude', 'smt']],
```
Comment (boegel) [inline easybuild/easyconfigs/m/Maude/Maude-3.1-GCCcore-10.2.0.eb:40]: @branfosj Also check for `bin/maude` here?

```suggestion
    'files': ['bin/maude'] + ['share/%s.maude' % x for x in ['file', 'linear', 'prelude', 'smt']],

**Review by boegel — CHANGES_REQUESTED** [review]

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3580.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/7e062282b17cd4a4d6609df994b63f46 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2601.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/0bec47866d4df0e0acea96ca043709e3 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @branfosj!

**Pull request merged**